samples/memory/sram_regions/boards/mps2_an385.overlay

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "dt_mpu_attr.h"
+
+/ {
+	/delete-node/ memory@20000000;
+
+	chosen {
+		zephyr,sram-no-cache = &sram_no_cache;
+	};
+
+	sram0: memory@20000000 {
+		compatible = "mmio-sram";
+		reg = <0x20000000 0x200000>;
+	};
+
+	sram_attr_0: memory@20200000 {
+		compatible = "mmio-sram";
+		reg = <0x20200000 0x100000>;
+		mpu-attr = <DT_SRAM_ATTR(DT_1MB)>;
```

Comment:
This looks for me that `dt_mpu_attr.h` header belongs somewhere else as into samples directory or am I missing something?

Reply:
Correct. This is a POC so I wanted to keep everything in one place.